solvers.tex

```diff
@@ -0,0 +1,127 @@
+%% Solvers in SymPy
+
+
+SymPy has a solvers module to solve equations symbolically. There are two
+submodules to solve algebraic equations in SymPy, referred to as old solve
```

Comment:
In SymPy the `solve` function is referred to as old solve, in the documentation of [solveset](http://docs.sympy.org/latest/modules/solvers/solveset.html) (the new solve), so, I think its a good idea to use same convention, isn't it?